index.js

```diff
@@ -84,11 +91,14 @@ module.exports = class MirrorDrive {
         continue
       }
 
+      const transformers = this.transformers.map((t) => t()).filter(Boolean)
```

Comment:
* `t` -> `transformer`
* `const stream = transformer()`
* if return value is null it should be a PassThough
* instead of map filter, `reduce((...) => {...}, [])` and then throw if if the return value is not a readable/writable stream 
  * can't assume streamx, could also be node streams
  *  so duck-type - look for push, write etc 
  *  throw if transformer return value is invalid
  * throw if transformer is not a function
  * add a test with invalid transformer